commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java

```diff
@@ -49,6 +50,8 @@ public void beforeEach(TestInfo testInfo) {
     Screenshots.startContext(testInfo.getTestClass().get().getName(), testInfo.getTestMethod().get().getName());
 
     openPlatform(testInfo);
+
+    Configuration.pageLoadStrategy="normal";
```

Comment:
Already part of another PR, should be removed.

Reply:
yes it is removed now, i added it to compare the execution time and I forgot to remove it